### PR TITLE
Make colors a bit nicer

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Didn't test, just my take on the colors

### CODE REVIEW

File: mediawiki_matrix_bot/__init__.py [hunk 1]
```diff
@@ -104,11 +104,11 @@ def format_data(obj: Dict[str, Any], baseurl: str, udpinput: bool = False) -> st
...
 
 
-    return  color('[[','gray') + bold(color(title,'yellow')) + color(']]','gray')  + \
+    return  color('[[ ','gray') + bold(color(title,'green')) + color(' ]]','gray')  + \
          " " + color(flag,'red') + \
         f" {url} {color('*','red')}"  + \
-        f" {color(user,'lightgreen')} {color('*','red')}" + \
-        f" {bold(diff_length)} {color(comment,'cyan')}"
+        f" {color(user,'yellow')} {color('*','red')}" + \
+        f" {color(diff_length, 'green')} {comment)}"
```
Comment (makefu) [inline mediawiki_matrix_bot/__init__.py:111]: syntax error, {comment)} -> {comment}

Comment (SuperSandro2000) [reply mediawiki_matrix_bot/__init__.py:111]:
```suggestion
        f" {color(diff_length, 'green')} {comment}"
```

**Comment by makefu** [comment]
incorporated in #4

**Comment by makefu** [comment]
thanks!

**Pull request closed without merging**